pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -452,9 +452,9 @@ func (configgen *ConfigGeneratorImpl) buildInboundClusters(cb *ClusterBuilder, p
 			if port, err = strconv.Atoi(hostPort); err != nil {
 				continue
 			}
-			if hostIP == model.PodIPAddressPrefix {
+			if hostIP == model.PodIPAddressPrefix || hostIP == model.PodIPv6AddressPrefix {
 				endpointAddress = cb.proxyIPAddresses[0]
```

Comment:
cb.proxyIPAddresses[0] can be ipv4 address when PodIPv6AddressPrefix = PodIPv6AddressPrefix

Reply:
okay, I can find the corrected IP for endpointAddress, submitted again. Thanks!